website/aws.erb

```diff
@@ -140,6 +140,9 @@
                         <li<%= sidebar_current("docs-aws-datasource-kms-secret") %>>
                             <a href="/docs/providers/aws/d/kms_secret.html">aws_kms_secret</a>
                         </li>
+                        <li<%= sidebar_current("docs-aws-datasource-nat-gateway") %>>
```

Comment:
Can you remove the extra plus signs on the lines below?

Reply:
They have been removed